packages/hygraph-cli/src/migrations/graphcommerce6to7.ts

```diff
@@ -303,5 +303,20 @@ export const graphcommerce6to7 = async (schema: Schema) => {
     'model',
   )
 
+  migrationAction(
+    schema,
+    'componentUnionField',
+    'create',
+    {
+      displayName: 'Conditions',
+      apiId: 'conditions',
+      parentApiId: 'ConditionOr',
+      componentApiIds: ['ConditionText', 'ConditionNumber'],
+      isList: true,
+    },
+    'ConditionOr',
+    'component',
+  )
+
```

Comment:
Please remove the migrations from older versions, this version has already been released and should not be modified